cli/compilers/ts.rs

```diff
@@ -638,24 +687,13 @@ async fn execute_in_thread(
   Ok(buf)
 }
 
-async fn execute_in_thread_json(
-  req_msg: Buf,
-  global_state: GlobalState,
-) -> JsonResult {
-  let msg = execute_in_thread(global_state, req_msg)
-    .await
-    .map_err(|e| OpError::other(e.to_string()))?;
-  let json_str = std::str::from_utf8(&msg).unwrap();
-  Ok(json!(json_str))
-}
-
-pub fn runtime_compile<S: BuildHasher>(
+pub async fn runtime_compile<S: BuildHasher>(
```

Comment:
Can you add a triple slash comment, since you're reworking this function, with some hints about what this function does?